sube/src/ws.rs

```diff
@@ -139,12 +169,23 @@ impl Backend<WS2> {
                             }
                         }
                     }
-                    Err(err) => {
-                        log::warn!("WS Error: {}", err);
+                    ewebsock::WsEvent::Error(e) => {
+                        log::warn!("WS error {}", &e);
+                    }
+                    ewebsock::WsEvent::Closed => {
+                        log::warn!("WS connection closed");
```

Comment:
```suggestion
                        log::info!("WS connection closed");
```
or debug ... it's not uncommon for connections to close